### DISCUSSION TITLE
Mojo::BaseUtil

### DISCUSSION THREAD

**Discussion by s1037989** [mojolicious/mojo#2195] [General]
Should `Mojo::BaseUtil` be documented in the Mojolicious docs API?

Also, the Changes file incorrectly refers to it as [Mojo::BaseUtils](https://github.com/mojolicious/mojo/blob/1c6b7f5fee4f04d119fe3c116deceaa73b0df46c/Changes#L9) (plural).

Another minor typo in the Changes file is the misspelling of "applications" as [applicartions](https://github.com/mojolicious/mojo/blob/1c6b7f5fee4f04d119fe3c116deceaa73b0df46c/Changes#L19).

**Comment by guest20** [comment]
From the PR that added that package, it's supposed to be private.

You're supposed to import `monkey_patch` from `Mojo::Util`

**Comment by kraih** [comment]
All modules belong in the API reference.
